mage_ai/orchestration/db/models.py

```diff
@@ -66,15 +66,16 @@ def save(self, commit=True) -> None:
                 self.session.rollback()
                 raise e
 
-    def update(self, **kwargs) -> None:
+    def update(self, commit=True, **kwargs) -> None:
```

Comment:
Can you put some docstring? So that it can be more transparent to others?
I guess rather than that its fine

Reply:
Thanks for the suggestions. We will add docstring to the methods.